src/shared/components/gene-search-panel/GeneSearchPanel.tsx

```diff
@@ -345,4 +361,18 @@ const getGroupedSearchMatches = (
   );
 };
 
+const NoResults = () => {
+  return (
+    <div className={styles.noResults}>
+      <p>
+        Found in <span className={styles.speciesCount}>0</span> species
+      </p>
+      <p className={styles.warning}>
+        Sorry, we can’t find this gene in any of your species in use
+      </p>
+      <p>Please use a different gene identifier</p>
+    </div>
+  );
+};
+
```

Comment:
No full stops for these paragraphs?

Reply:
Andrea doesn't do full stops 🤷‍♂️ 

XD: https://xd.adobe.com/view/763d10be-0183-4cea-afef-b9a35a54f1c3-236e/